00-vars.tf

```diff
@@ -1,6 +1,6 @@
 terraform {
   required_providers {
-    packet = "2.7.5"
+    packet = "~> 2.10.1"
```

Comment:
Noting that this takes advantage of the pessimistic version operator:

https://www.terraform.io/docs/configuration-0-11/terraform.html#specifying-a-required-terraform-version

Reply:
```
~>: pessimistic constraint operator. Example: for ~> 0.9, this means >= 0.9, < 1.0. Example: for ~> 0.8.4, this means >= 0.8.4, < 0.9
```